src/projector.ts

```diff
@@ -29,9 +29,17 @@ export interface ProjectorOptions extends ParentMixinOptions<Child>, EventedOpti
 	 *
 	 * Only applies if `autoAttach` is `true`
 	 */
+	// FIXME Is this used? What about replace?
 	append?: boolean;
 }
 
+export interface AttachOptions {
+	append?: boolean;
```

Comment:
Importing the enums seems annoying, so I prefer the second suggestion. If the options are optional though then `type` should be optional too (but default to `merge`).